Build/Scripts/runTests.sh

```diff
@@ -155,7 +155,9 @@ Options:
             - functional: PHP functional tests
             - lintTypoScript: TypoScript linting
             - lintPhp: PHP linting
+            - lintJs: JavaScript file linting. Set -n for dry-run.
             - lintJson: JSON linting
+            - lintStyle: CSS file linting. Set -n for dry-run.
```

Comment:
I'd expect this command to be named `lintCss` instead to be consistent with the other commands that have the language/file type after "lint".